packages/stg-devtools-evm-hardhat-v2/src/blacklistable/mixin.ts

```diff
@@ -44,6 +46,8 @@ export const BlacklistableMixin: IBlacklistable = {
         return {
             ...this.createTransaction(data),
             description: `Setting ${address} blacklist status to ${blacklisted}`,
+            functionName: 'setBlacklisted',
+            functionArgs: `address = ${address}\nblacklisted = ${blacklisted}`, // TODO confirm this pattern with Jan before proceeding to make this change in all files
```

Comment:
@janjakubnanista I have a devtools PR up [here](https://github.com/LayerZero-Labs/devtools/pull/991/files) that adds these 2 properties. While deploying stargate, caleb mentioned it would be best if we could always display:

1. contract name
  - I believe this can be accessed through `Omnipoint.contractName` but that is currently undefined so looking into it
2. function name and functionArgs
  - As you can see above i added both of these as a new property on OmniTransaction in the devtools PR. If you confirm this pattern is acceptable, I will do this for all sdks where transactions are created. I see that `functionArgs` clashes with `description` in some cases, so I will try to remove redundancy as much as possible when updating the other functions.